input/ig.xml

```diff
@@ -581,15 +573,15 @@
 		</resource>
 		<resource>
 			<reference>
-			<reference value="Patient/banks-mia-leanne"/>
+				<reference value="Patient/banks-mia-leanne"/>
 			</reference>
 			<name value="Patient - Mia LEANNE BANKS"/>
 			<description value="Mr. Li Wang"/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient"/>
 		</resource>
 		<resource>
 			<reference>
-			<reference value="Patient/howe-deangelo"/>
+				<reference value="Patient/howe-deangelo"/>
 			</reference>
 			<name value="Patient - Deangelo HOWE"/>
 			<description value="Mr. Li Wang"/>
```

Comment:
While we're changing examples, fix the description here

Reply:
Done